Dockerfile.assisted-service

```diff
@@ -25,9 +25,22 @@ RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service
 RUN cd build && python3 setup.py sdist --dist-dir /assisted-service-client
 
 # Create final image
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM centos:8
+
+# openshift-install requires this
+RUN yum install -y libvirt-libs && yum clean all
+
+ARG WORK_DIR=/data
+
+RUN mkdir $WORK_DIR && chmod 755 $WORK_DIR
```

Comment:
This needs to be `775` for us to be able to write here under the restricted scc, right?

Reply:
Good point. Are we doing any of the other related things? Looks like the Dockerfile still runs the process as root for example...

Reply:
What "other related things"? We can build as root an run as non-root. If we want to try to test an openshift-like scenario in kubernetes we could drop a bunch of capabilities and enforce a non-root uid in the deployment security context, but I don't think that would change the Dockerfile, right?